transcoder/basisu_transcoder.cpp

```diff
@@ -10311,17 +10315,14 @@ namespace basist
 					BASISU_DEVEL_ERROR("basisu_transcoder::transcode_image_level: transcode_slice() to ATC_RGB failed\n");
 				}
 				break;
+#endif
 			}
 			case transcoder_texture_format::cTFATC_RGBA:
 			{
 #if !BASISD_SUPPORT_ATC
             BASISU_DEVEL_ERROR("basisu_transcoder::transcode_image_level: ATC unsupported\n");
 				return false;
-#endif
-#if !BASISD_SUPPORT_DXT5A
```

Comment:
This was removed because earlier a test for `BASISD_SUPPORT_ATC` will `#error` if `BASISD_SUPPORT_DXT5A` is missing.